src/include/migraphx/op/quantizelinear.hpp

```diff
@@ -38,6 +38,7 @@ namespace op {
 struct quantizelinear
 {
     std::string name() const { return "quantizelinear"; }
+    std::optional<migraphx::shape> output_shape;
```

Comment:
This should be `std::optional<migrapx::shape::type_t>`, we only need the output data type. We dont need to the dimensions or strides as we will get that from the inputs.

Reply:
Done.